packages/core/src/Stores/client-store.js

```diff
@@ -2718,6 +2721,28 @@ export default class ClientStore extends BaseStore {
         this.should_show_effortless_login_modal = should_show_effortless_login_modal;
     }
 
+    setPasskeysStatusToCookie(status) {
+        let domain = /deriv.com/.test(window.location.hostname) ? URLConstants.derivHost : window.location.hostname;
+
+        if (/deriv.dev/.test(window.location.hostname)) {
+            //set domain for dev environment (FE deployment and login page on qa-box)
+            domain = 'deriv.dev';
+        }
+
+        const expirationDate = new Date();
+        expirationDate.setFullYear(expirationDate.getFullYear() + 1); // Set to expire in 1 year
+
+        const is_available = status === 'available';
```

Comment:
should we move this available to a constant file?

Reply:
@suisin-deriv , very valid question. I think we can skip adding new constant as there is just two statuses possible - 'available' and. 'not available'
when we use this function in tsx files, TS helps us to pass valid data.